supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiJobPaths.scala

```diff
@@ -13,6 +13,7 @@ object PipelinesApiJobPaths {
   val GcsTransferLibraryName = "gcs_transfer.sh"
   val GcsLocalizationScriptName = "gcs_localization.sh"
   val GcsDelocalizationScriptName = "gcs_delocalization.sh"
+  val DrsLocalizationManifestName = "drs_manifest"
```

Comment:
From other parts of the code, it seems like this is a CSV? Extensionless files seem like a mild missed opportunity for self-documentation.
```suggestion
  val DrsLocalizationManifestName = "drs_manifest.csv"
```